### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by lynetcha** [ThibaultGROUEIX/AtlasNet#9]
I ran the train_AE_AtlasNet script followed by run_AE_AtlasNet. The results do not seem quite right. 

AtlasNet output is below
![image](https://user-images.githubusercontent.com/11138935/40856605-49386d98-658d-11e8-9301-37685ecac796.png)

Ground-truth
![image](https://user-images.githubusercontent.com/11138935/40856481-e570a2f8-658c-11e8-8d21-260032d843d2.png)

**Comment by ThibaultGROUEIX** [comment]
hi ! indeed this is not right. can you include your training and testing loss plots ?

**Comment by lynetcha** [comment]
Here is the visdom visualization

![image](https://user-images.githubusercontent.com/11138935/40857397-e69b01de-658f-11e8-88f0-8a976b057d66.png)

**Comment by lynetcha** [comment]
I just saw the issue about the pytorch version. Maybe that is my problem. I am using 0.3.1 version. I will try 0.4.0 and get back to you. Thanks.

**Comment by lynetcha** [comment]
I am actually using python 3.6. Judging by the table you provided, the Chamfer loss doesn't work with 3.6 and pytorch 0.4, is that correct?

**Comment by ThibaultGROUEIX** [comment]
definitly the pytorch version. your curves should look as in the readme. good luck !

**Comment by ThibaultGROUEIX** [comment]
apparently not (i don'r remember it exactly). it boils down to the chamfer distance code which is not compatible to every pytorch version. why not install a conda env with python 2.7?

**Comment by lynetcha** [comment]
I tried conda python 2.7 and pytorch 1.12 and training was stable. Thanks!

**Comment by lynetcha** [comment]
@ThibaultGROUEIX It seems that removing the batch normalization layers solves the instability issues during training due to pytorch versions. I was able to train the model without BN in python 3.6, pytorch 0.3.1 and achieve 1.81 loss for the autoencoder after 200 epochs.
![image](https://user-images.githubusercontent.com/11138935/41486382-23b5d198-7099-11e8-8f52-c174ebf99ec8.png)

**Comment by ThibaultGROUEIX** [comment]
unfortunatly, i can't provide support if you change the architecture but thanks for linking Chamfer error to batch norm!
regards, Thibault

**Comment by ThibaultGROUEIX** [comment]
actually i am not clear on if 1.81 is an issue or a success in your case. what error are you reporting ?

**Comment by lynetcha** [comment]
It’s not an error. I’m just pointing out a potential workaround for those who may want to use the architecture without reverting back to an old pytorch version.

**Comment by ThibaultGROUEIX** [comment]
Great ! Thanks for clarifying !
